### PR TITLE
Travis: Fix ccache on debug builds

### PR DESCRIPTION
**Pull request by peterbell10**
Debug builds seem to be really slow right now. Wondering if `ccache` isn't working.

### CODE REVIEW

**Comment by bearbin** [comment]
Ccache is working, it's just not getting any cache hits. Cache size appears to be near the set limit, but that shouldn't reduce the hits to near zero really. Maybe the cache is only getting propagated to the next build from the release build somehow?

**Comment by bearbin** [comment]
It's fixed! Reasonably fast builds at last :)

**Comment by peterbell10** [comment]
For some reason the debug builds had a 500 MB cache limit so the cache was being thrashed,

File: travisbuild.sh [hunk 1]
```diff
@@ -11,16 +11,24 @@ if [ `which ccache` ]; then
...
 cmake . -DBUILD_TOOLS=1 -DSELF_TEST=1 ${CACHE_ARGS};
 
 echo "Building..."
-cmake --build . -- -j 2;
-ctest -j 2 -V;
+cmake --build . -j 2
+
+if [ `which ccache` ]; then
+		echo "Built with ccache, outputting cache stats..."
+		ccache -s
```
Comment (tigerw) [inline travisbuild.sh:25]: waaaait what are these, TWO tabs???

**Pull request merged**